libc/docs/dev/config_options.rst

```diff
@@ -123,7 +121,7 @@ should convert the CMake config options to appropriate compiler and/or linker
 flags. Those compile/link flags can be used in listing the affected targets as
 follows:
 
-.. code-block::
+.. code-block:: none
```

Comment:
```suggestion
.. code-block:: cmake
```

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/code_blocks.html says that `pygments` is what's used for syntax highlighting.

https://pygments.org/languages/ claims that `cmake` is supported. Can you use that here rather than none?